### PR TITLE
[release-4.8] Bug 2039538: Add custom profile metric for Node Tuning Operator to telemetry

### PR DESCRIPTION
**Pull request by dagrayvid**
To help prioritize future plans for NTO, we want to better understand how it is being used. One way we can do this is by sending metrics through telemetry to see how often NTO is being used to manage custom TuneD profiles on nodes. This is available on 4.10, and 4.9. 

* [ ] I added CHANGELOG entry for this change.
* [x] No user facing changes, so no entry in CHANGELOG was needed.

### CODE REVIEW

File: manifests/0000_50_cluster-monitoring-operator_04-config.yaml [hunk 1]
```diff
@@ -355,6 +355,10 @@ data:
     # 99th percentile of the backend commit duration.
     - '{__name__="instance:etcd_disk_backend_commit_duration_seconds:histogram_quantile",quantile="0.99"}'
     #
+    # owners: (https://github.com/openshift/cluster-node-tuning-operator)
```
Comment (jan--f) [inline manifests/0000_50_cluster-monitoring-operator_04-config.yaml:358]: Not sure a github repo should be a metric owner here. Can we find a team like for the other owner entries?

**Review by jan--f — COMMENTED** [review]
The generally looks good, pending one nit.

File: manifests/0000_50_cluster-monitoring-operator_04-config.yaml (hunk 1 above)
Comment (dagrayvid) [reply manifests/0000_50_cluster-monitoring-operator_04-config.yaml:358]: Thanks Jan, for the review. There were some other owners pointing to a github repo in this file on the release-4.10 branch (from where it was backported). I will update it here to keep it more in line with the other entries.

**Comment by dagrayvid** [comment]
@jan--f, I replaced the "team" with our github team from the OCP org. Thanks!

File: manifests/0000_50_cluster-monitoring-operator_04-config.yaml (hunk 1 above)
Comment (jan--f) [reply manifests/0000_50_cluster-monitoring-operator_04-config.yaml]: Thanks, I'll go after that other entry right away :wink:

**Review by jan--f — APPROVED** [review]
/lgtm

**Comment by dagrayvid** [comment]
/retest

@jan--f , this PR will also need the backport-risk-assessed label, if it's ready to go from your side.

**Pull request merged**